### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by sooth-sayer** [avocode/avocode#570]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.5
Source UID: 135687
```
Error: ENOENT: no such file or directory, lstat '/Users/che/.avcd/userdata/135687/documents/9c9950bfcb9e4f2c9e8bd09e65196f24/images/.archive/1151117-2508-a4m23x/all.png'
    at Error (native)
```

**Comment by jakubzitny** [comment]
Hi, thanks for the report. We know about this issue and we will take care of it in next releases. However, some details about how did you get this error would help.

This is related to zlib problems mentioned in #447. I'll close this issue, let's continue the discussion there.

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
